PS8lite.Rmd

```diff
@@ -187,23 +188,54 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
+#define cross-validation settings: 10-fold CV 
+fit_control <- trainControl(method = "cv", number = 10)
```

Comment:
Hi, everything looks good. Here is an example comment that I would leave as part of a “code review” if this were real life. Then you would make the requested changes to this PR and then I would merge your changes into my master copy of the repo. I won't be doing that last step in this case.